it/test/helpers/GenericStubMethods.scala

```diff
@@ -22,10 +22,8 @@ trait GenericStubMethods extends CustomMatchers {
 
   def isAuthorisedUser(authorised: Boolean): Unit = {
     if (authorised) {
-      Given("I wiremock stub an isAuthorisedUser user response")
```

Comment:
not sure why we are dropping Given statements here, its a small thing, but I don't see reason to drop these